scripts/generateServiceConfigs/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
The generator is nice, but this is a one time deal and we cannot post it in an OSS repo anyway, so please remove it from this PR